### DISCUSSION TITLE
Add an option to sync a fork repository

### DISCUSSION THREAD

**Discussion by Zerotask** [community/community#21620] [New to GitHub]
<p>It would be great if you’d have an option to sync your fork with the original repository. This way it would be much easier for beginners to contribute to open source software.</p>
<p>At the moment GItHub just provides 3 help pages which are really good and helpful, but I think this could be automated optionally (e. g. with a button “Sync fork”)</p>

**Comment by canuckjacq** [comment]
<p>Hi <a class="mention" href="/zerotask">@Zerotask</a>  - this is a popular feature request. I’ve added your request to the list! :crossed_fingers:</p>

**Comment by adlai** [comment]
<p>I’m fairly certain it was once possible to workaround that feature’s absence, by enabling rebase-only, and then sending yourself a pull request, although I haven’t tried this recently.</p>

**Comment by airtower-luna** [comment]
<p>That’s not going to work, because a rebase merge recreates all the commits with the person doing the merge as the committer, so the commits aren’t the same after.</p>

**Comment by canuckjacq** [comment, marked as answer]
<p>:ship: We have an update!</p>
<aside class="onebox twitterstatus">
  <header class="source">
      <a href="https://twitter.com/github/status/1390382527588798477" target="_blank" rel="noopener">twitter.com</a>
  </header>
  <article class="onebox-body">
    <img src="https://user-images.githubusercontent.com/11161530/181065865-70d2438d-a53f-41f7-b5a7-5fdb5e69aa3e.png" class="thumbnail onebox-avatar" width="400" height="400">

<h4><a href="https://twitter.com/github/status/1390382527588798477" target="_blank" rel="noopener">GitHub (github)</a></h4>

<div class="tweet"> Is your fork behind? You can now sync from the parent repo with just a single click! 🖱💥 https://t.co/4Zx6OwVTxO</div>

<div class="date">
  <a href="https://twitter.com/github/status/1390382527588798477" target="_blank" rel="noopener">12:06 PM - 6 May 2021</a>
    <span class="like">
      <svg viewbox="0 0 512 512" width="14px" height="16px" aria-hidden="true">
        <path d="M462.3 62.6C407.5 15.9 326 24.3 275.7 76.2L256 96.5l-19.7-20.3C186.1 24.3 104.5 15.9 49.7 62.6c-62.8 53.6-66.1 149.8-9.9 207.9l193.5 199.8c12.5 12.9 32.8 12.9 45.3 0l193.5-199.8c56.3-58.1 53-154.3-9.8-207.9z"></path>
      </svg> 7.9K
    </span>
    <span class="retweet">
      <svg viewbox="0 0 640 512" width="14px" height="16px" aria-hidden="true">
        <path d="M629.657 343.598L528.971 444.284c-9.373 9.372-24.568 9.372-33.941 0L394.343 343.598c-9.373-9.373-9.373-24.569 0-33.941l10.823-10.823c9.562-9.562 25.133-9.34 34.419.492L480 342.118V160H292.451a24.005 24.005 0 0 1-16.971-7.029l-16-16C244.361 121.851 255.069 96 276.451 96H520c13.255 0 24 10.745 24 24v222.118l40.416-42.792c9.285-9.831 24.856-10.054 34.419-.492l10.823 10.823c9.372 9.372 9.372 24.569-.001 33.941zm-265.138 15.431A23.999 23.999 0 0 0 347.548 352H160V169.881l40.416 42.792c9.286 9.831 24.856 10.054 34.419.491l10.822-10.822c9.373-9.373 9.373-24.569 0-33.941L144.971 67.716c-9.373-9.373-24.569-9.373-33.941 0L10.343 168.402c-9.373 9.373-9.373 24.569 0 33.941l10.822 10.822c9.562 9.562 25.133 9.34 34.419-.491L96 169.881V392c0 13.255 10.745 24 24 24h243.549c21.382 0 32.09-25.851 16.971-40.971l-16.001-16z"></path>
      </svg> 1.8K
    </span>
</div>

  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>

**Comment by Zerotask** [comment]
<p>Thanks a lot. I just tested it and it works like a charm. It’s so much easier and also faster than the “classic” way. :slight_smile:</p>

**Comment by canuckjacq** [comment]
<p>That’s so great to hear!</p>
